catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -61,7 +61,7 @@
         <bundle>mvn:ddf.catalog.core/catalog-core-resourcesizeplugin/${project.version}</bundle>
 
         <configfile finalname="/data/solr/metacard_cache/conf/solrconfig.xml">mvn:ddf.platform.solr/platform-solr-server-standalone/${project.version}/xml/solrconfig</configfile>
-        <configfile finalname="/data/solr/metacard_cache/conf/schema.xml">mvn:ddf.platform.solr/platform-solr-server-standalone/${project.version}/xml/schema</configfile>
+        <configfile finalname="/data/solr/metacard_cache/conf/schema.xml">mvn:ddf.platform.solr/platform-solr-server-standalone/${project.version}/xml/schema_metacard</configfile>
```

Comment:
We need to consolidate back to a single schema.  I would suggest removing the current schema.xml and rename schema_metacard.xml to schema.xml.  The persistence store will need more work to fully support multiple values.  You can use `getFirstValue` in PersistenceStoreImpl for now with a ticket to fully support multiple values.

https://lucene.apache.org/solr/5_3_1/solr-solrj/org/apache/solr/common/SolrDocument.html#getFirstValue(java.lang.String)